### PR TITLE
Re-design UI to show both configurations (#153)

### PR DESCRIPTION
**Pull request by alxdavids**
Summary:

- Show both configurations at the same time (demarcate which is active)
- Remove "Get More Passes" button
- Replace with `mouseover` characteristics to get more passes.

Images showing proposed changes (feedback welcome!):

Old UI:

image

New UI:

image
image

### CODE REVIEW

File: addon/popup/bc-plugin.js [hunk 1]
```diff
@@ -91,31 +87,51 @@ function outputRedemption(v, s1, s2) {
...
-    newCount.appendChild(document.createTextNode(tokLen));
-    document.getElementById("stored").appendChild(newCount);
+        stored.onmouseover = () => {
+            const passesText = document.getElementById(`passtext-${ele.id}`);
+            passesText.textContent = "Get more passes!";
+        };
+        stored.onmouseleave = () => {
+            const passesText = document.getElementById(`passtext-${ele.id}`);
+            passesText.textContent = `${ele.name} ${active}`;
+        }
```
Comment (Lekensteyn) [inline addon/popup/bc-plugin.js:131]: Missing semicolon.

File: addon/popup/bc-plugin.js [hunk 2]
```diff
@@ -15,40 +15,37 @@ if (background) {
...
+                UpdatePopup();
+            })
         }
     });
 
     // this allows the client to generate a redemption token for CF API.
     document.getElementById("redeem").addEventListener("click", () => {
         if (background) {
-            tokLen = background.countStoredTokens();
+            tokLen = background.countStoredTokens(background.getConfigId());
```
Comment (armfazh) [inline addon/popup/bc-plugin.js:48]: The extension shows an error when I clicked on "Log redemption console"

>  Uncaught TypeError: background.getConfigId is not a function

**Comment by armfazh** [comment]
How do one can toggle between configurations?

**Comment by alxdavids** [comment]
@armfazh The redemption button only corresponds to the CF configuration as it is only intended to be used with the redemption API at privacypass.cloudflare.com/api/redeem. I've reworded the text to make it a bit clearer.

**Pull request merged**